src/openfe_analysis/utils/multistate.py

```diff
@@ -135,9 +194,15 @@ def _get_unitcell(dataset: nc.Dataset, replica_index: int, frame_num: int):
 
     Returns
     -------
-    Tuple[lx, ly, lz, alpha, beta, gamma]
+    Optional[Tuple[lx, ly, lz, alpha, beta, gamma]]
         Unit cell lengths and angles in angstroms and degrees.
+        If box_vectors are masked (i.e. they were not stored at this frame),
+        will return ``None``.
     """
+    # Case: no box_vectors were stored at this frame
+    if dataset.variables['box_vectors'][frame_num][replica_index].mask.all():
```

Comment:
Could you add a test in `test_multistate` with the skipped netcdf that tries to access the positions and unitcell at frame 1 and gets a None?